lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -98,14 +98,16 @@ async function runA11yChecks() {
 }
 /* c8 ignore stop */
 
+/** @implements {LH.Gatherer.FRGathererInstance<'AppCacheManifest'>} */
```

Comment:
these annotations are necessary to parameterize and matchup the `meta.dependencies` with the `snapshot`/`timespan` method types